src/lib/config.js

```diff
@@ -11,13 +13,65 @@ function isRunningTests () {
      process.argv[1].endsWith('mocha'))
 }
 
+function useTestConfig () {
+  return !Config.castBool(process.env.UNIT_TEST_OVERRIDE) && isRunningTests()
+}
+
 function generateDefaultPairs (ledgers) {
   return Utils.getPairs(ledgers).map((pair) => {
     return pair.map(desc => desc.currency + '@' + desc.ledger)
   })
 }
 
+function parseCredentialsEnv () {
+  return JSON.parse(Config.getEnv(envPrefix, 'CREDENTIALS') || '{}')
+}
+
+function parseCredentials () {
+  const credentialsEnv = parseCredentialsEnv()
+
+  return _.reduce(credentialsEnv, (parsed, credentials, ledger) => {
```

Comment:
I can refactor this in the future, as it would require a major version bump for lodash